bootstrap/pkg/kfapp/ksonnet/ksonnet.go

```diff
@@ -447,16 +447,18 @@ func (ksApp *ksApp) initKs() error {
 	}
 	ksApp.KsEnvName = KsEnvName
 	k8sSpec := ksApp.Spec.ServerVersion
+	host := "127.0.0.1"
 	if k8sSpec == "" {
-		k8sSpec = kftypes.GetServerVersion(kftypes.GetClientset(kftypes.GetConfig()))
```

Comment:
Shall we just throw exception on ```k8sSpec == ""```?
If we cache spec in kubeflow repo it should never be empty?
https://github.com/kubeflow/kubeflow/pull/2765/files